celesta-test/src/test/java/ru/curs/celesta/dbutils/adaptors/AbstractAdaptorTest.java

```diff
@@ -1683,24 +1681,32 @@ public void testCompareStrings() throws Exception {
         assertEquals(1, comparisonResult);
     }
 
+
     @Test
     void testVarcharFieldEnumeratorCollation() throws Exception {
-        List<String> data = dba.selectStaticStrings(VarcharFieldEnumerator.CHARS, "\"id\"", "\"id\" ASC");
-        VarcharFieldEnumerator varcharFieldEnumerator = new VarcharFieldEnumerator(dba, 1);
-
-        BigInteger prevOrder = BigInteger.valueOf(-1);
-
-        for (int i = 0; i < data.size(); ++i) {
-            String ch = data.get(i);
-            varcharFieldEnumerator.setValue(ch);
-
-            BigInteger currentOrder = varcharFieldEnumerator.getOrderValue();
-            assertTrue(prevOrder.longValue() < currentOrder.longValue());
-            prevOrder = currentOrder;
-        }
+        List<String> test = Arrays.asList(
+                "'", "-", "–", "—", " ", "!", "\"", "#", "$", "%", "&", "(", ")",
```

Comment:
`VarcharFieldEnumerator.CHARS` is already contains a lot of chars? Why is it needed here?

Reply:
This is what typically `VarcharEnumerator` is going to send to `selectStaticStrings`.

We decoupled this test from Lyra's `VarcharEnumerator` , but for 'good test' I just copy-pasted the `VarcharEnumerator`'s request: if it is not failing on tests, it will work on real databases.